docs/user/BuildLocally.md

```diff
@@ -43,6 +43,37 @@ make gui_final
 
 ![gcd_final.webp](../images/gcd_final.webp)
 
+## Using Pre-built Binaries
+
+You can download pre-built binaries with self contained dependencies
+included from the Precision Innovations github repository
```

Comment:
Please add verbiage like this to make sure its known that PII is providing this binary as a service to the community vs it coming from UCSD or the project.

"Thanks to Precision Innovations (www.precisioninno.com) for providing and supporting this daily release of prebuilt installers to the community."

Reply:
Added at the end of the section.